### PR TITLE
doc/rados: edit t-mon "common issues" (2 of x)

### PR DESCRIPTION
**Pull request by zdover23**
Edit the second part of the section "Most Common Monitor Issues" in doc/rados/troubleshooting/troubleshooting-mon.rst.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Comment by zdover23** [comment]
Follows https://github.com/ceph/ceph/pull/54413.

File: doc/rados/troubleshooting/troubleshooting-mon.rst [hunk 1]
```diff
@@ -213,31 +213,38 @@ you should be seeing something similar to::
...
-  ``monmap`` known to the monitor: check if the other monitor's locations
-  match reality. If they don't, jump to
-  `Recovering a Monitor's Broken monmap`_; if they do, then it may be related
-  to severe clock skews amongst the monitor nodes and you should refer to
-  `Clock Skews`_ first, but if that doesn't solve your problem then it is
-  the time to prepare some logs and reach out to the community (please refer
-  to `Preparing your logs`_ on how to best prepare your logs).
+**What does it mean if  a Monitor's state is ``probing``?**
+
+  If ``ceph health detail`` returns the information that a Monitor's state is
```
Comment (anthonyeleven) [inline doc/rados/troubleshooting/troubleshooting-mon.rst:218]: S/returns the information/shows/

File: doc/rados/troubleshooting/troubleshooting-mon.rst [hunk 2]
```diff
@@ -213,31 +213,38 @@ you should be seeing something similar to::
...
+  |---| this means that if two out of three Monitors in the cluster are
+  ``down``, the one remaining Monitor stays in the ``probing``  state
+  indefinitely until you bring one of the other monitors up.
+
+  If quorum has been established, then the Monitor daemon should be able to
+  find the other Monitors quickly, as long as they can be reached. If a Monitor
+  is stuck in the ``probing`` state and you have exhausted the procedures above
+  that describe the troubleshooting of communications between the Monitors,
+  then it is possible that the problem Monitor is trying to reach the other
+  Monitors on a wrong address. ``mon_status`` outputs the ``monmap`` that is
```
Comment (anthonyeleven) [inline doc/rados/troubleshooting/troubleshooting-mon.rst:237]: S/on/at/

File: doc/rados/troubleshooting/troubleshooting-mon.rst [hunk 3]
```diff
@@ -213,31 +213,38 @@ you should be seeing something similar to::
...
+  known to the monitor: determine whether the other Monitors' locations as
+  specified in the ``monmap`` match the locations of the Monitors in the
+  physical world. If they do not, see `Recovering a Monitor's Broken monmap`_.
```
Comment (anthonyeleven) [inline doc/rados/troubleshooting/troubleshooting-mon.rst:240]: S/physical world/network/

**Review by anthonyeleven — APPROVED** [review]

File: doc/rados/troubleshooting/troubleshooting-mon.rst (hunk 3 above)
Comment (zdover23) [reply doc/rados/troubleshooting/troubleshooting-mon.rst:240]: Much better.

File: doc/rados/troubleshooting/troubleshooting-mon.rst (hunk 1 above)
Comment (zdover23) [reply doc/rados/troubleshooting/troubleshooting-mon.rst:218]: Much better.

Accepted.

File: doc/rados/troubleshooting/troubleshooting-mon.rst (hunk 2 above)
Comment (zdover23) [reply doc/rados/troubleshooting/troubleshooting-mon.rst:237]: Accepted.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/54421 - Reef backport
https://github.com/ceph/ceph/pull/54422 - Quincy backport

**Pull request merged**